src/pages/validator/amplifier/verifier-onboarding.mdx

```diff
@@ -1,197 +1,147 @@
-# Become an Amplifier Verifier (Worker)
+# Become an Amplifier Verifier
 
 import { Callout } from "/src/components/callout"
 
 <Callout>
 The Axelar Virtual Machine (AVM) and Amplifier are currently under active development, so these instructions are likely to change. Please check back frequently for updates.
 </Callout>
 
-By running a **verifier** (also called a **worker**) for a chain integration with Axelar, you will be responsible for voting on the validity of transactions on the Axelar network for a given chain.
+By running a **verifier** for a chain integration with Axelar, you will be responsible for voting on the validity of transactions on the Axelar network and signing command batches for a given chain.
 
 ## Prerequisites
 
-1. Pull the **`tofnd`** and `ampd` image from [Docker Hub](https://hub.docker.com/):
-    ```bash
-    docker pull haiyizxx/tofnd:latest
-    docker pull haiyizxx/ampd:v0.0.2
-    ```
-1. Create an `ampd` folder in your working directory, download [`config.toml`](https://file.notion.so/f/f/8f1eb2f6-501d-4ae8-bd56-d35b21d3750f/a9bc03f8-a772-41bb-ad17-4f1b650f18ca/config.toml?id=6a8ec351-527a-467e-ab81-abc7ee8a9950&table=block&spaceId=8f1eb2f6-501d-4ae8-bd56-d35b21d3750f&expirationTimestamp=1705082400000&signature=qeG1WvtWM640SjvpYwg3ZMcBgEpCbIq2gUO1diGYbVQ&downloadName=config.toml), and put it in `ampd` folder.
+1. Run `tofnd`. 
 
-## Onboard as a verifier
+If you do not have it, you can pull and run the latest with the following:
 
-### Run `tofnd`
+```bash
+docker pull haiyizxx/tofnd:latest
+docker run -p 50051:50051 --env MNEMONIC_CMD=auto --env NOPASSWORD=true -v tofnd:/.tofnd haiyizxx/tofnd:latest
```

Comment:
```suggestion
docker pull axelarnet/tofnd:v0.10.1
docker run -p 50051:50051 --env MNEMONIC_CMD=auto --env NOPASSWORD=true -v tofnd:/.tofnd axelarnet/tofnd:v0.10.1
```
shouldn't use haiyi's image. feel free to ping other folks in core team/devops to find this info